game_eggs/eft/egg-singleplayer-tarkov

```diff
@@ -0,0 +1,62 @@
+        "startup": "{\r\n    \"done\": \"Server is running, do not close while playing SPT, Happy playing!!\"\r\n}",
+        "logs": "{}",
+        "stop": "^C"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\r\n# Singleplayer Tarkov Server Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\n\r\n# Working directory for build requirements\r\ncd $HOME\r\n\r\n# Install git-lfs python3\r\napt update && apt install -yq git-lfs python3 unzip jq\r\n\r\n# Install Node Version Manager and NodeJS\r\ngit clone https:\/\/github.com\/nvm-sh\/nvm.git $HOME\/.nvm\r\nif [ -z \"$NODE_VERSION\" ] || [ \"$NODE_VERSION\" == \"LTS\" ]; then\r\n  NODE_VERSION=$(curl -sSL https:\/\/nodejs.org\/download\/release\/index.json | jq -r '.[] | select(.lts!=false) | .version' | head -n1)\r\nfi\r\n. $HOME\/.nvm\/nvm.sh && nvm install $NODE_VERSION\r\n\r\n# Clone the SPT AKI repo\r\ngit clone https:\/\/dev.sp-tarkov.com\/SPT-AKI\/Server.git server\r\ncd $HOME\/server && git checkout $SPT_VERSION\r\n\r\n# Obtaining server files\r\ncd $HOME\/server\/project\/\r\ngit-lfs fetch --all && git-lfs pull\r\n\r\n# Install npm dependencies and build\r\n. $HOME\/.nvm\/nvm.sh && npm install && npm run build:release\r\nmv $HOME\/server\/project\/build\/* \/mnt\/server\/\r\n\r\n# Obtain SIT Coop mod\r\ngit clone https:\/\/github.com\/stayintarkov\/SIT.Aki-Server-Mod \/mnt\/server\/user\/mods\/SITCoop\r\ncd \/mnt\/server\/user\/mods\/SITCoop && git checkout $SIT_VERSION",
+            "container": "ghcr.io\/pterodactyl\/installers:debian",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "SPT-AKI Version",
+            "description": "Version of the SPT-AKI version to use.\r\nUse \"latest\" for the latest tag on the master branch.\r\n\r\nSPT-AKI: https:\/\/dev.sp-tarkov.com\/SPT-AKI\/Server",
+            "env_variable": "SPT_VERSION",
+            "default_value": "40b999d04c68f1f52ab152d163c086a1c50f489b",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string",
+            "field_type": "text"
+        },
+        {
+            "name": "SITCoop Version",
+            "description": "Version of the SIT co-op mod for the server to use, by git hash.\r\n\r\nSITCoop: https:\/\/github.com\/stayintarkov\/SIT.Aki-Server-Mod",
+            "env_variable": "SIT_VERSION",
+            "default_value": "022caac252205e1b9ca30b3610884ad1b05d0dc1",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string",
+            "field_type": "text"
+        },
+        {
+            "name": "NodeJS Version",
+            "description": "Version of NodeJS to use to build the server.\r\nUse \"LTS\" for latest LTS version, or \"20.10.0\" for the recommended version.",
```

Comment:
This should not be needed. If the game container needs node then use a node image.

Reply:
Node is only used to build the server in the install container, the game container doesn't need it. Should it be hardcoded to a working node version, or left as an environment variable?

Reply:
hardcode it or set it as a ENV variable that is only present in the install script, and install it the offical way not with nvm

(you can always look at the build docs how pterodactyl does this)

Reply:
I updated it to use NodeSource method to install Node.js [as outlined here](https://nodejs.org/en/download/package-manager#debian-and-ubuntu-based-linux-distributions). Is this what you mean by official?
Either way, I'll hardcode it to the LTS version in the install script.

Reply:
yes that is what I meant. thats fine